macros/unpack/get_node_values.sql

```diff
@@ -5,12 +5,15 @@
 {%- macro default__get_node_values() -%}
 
     {%- if execute -%}
+    {% set re = modules.re %}
```

Comment:
Do you still need this now that you've created the separate `set_is_excluded` macro?

Reply:
Eagle eye! Let me fix that!